tests/test_manager/test_utils.py

```diff
@@ -37,26 +37,26 @@ def _process_exception():
     raise ValueError("Expected")
 
 
-def _process_return_value():
-    return RETURN_VALUE
+def _process_return_value(value_to_return):
+    return value_to_return
 
 
 def test_process_run_in_venv_timeout_error():
     """Test timeout error raised for process running too long."""
     with TemporaryDirectory() as tmp_dir:
         with pytest.raises(TimeoutError):
-            run_in_venv(tmp_dir, _process_long, timeout=3)
+            run_in_venv(tmp_dir, _process_long, timeout=20)
```

Comment:
Introduce a constant DEFAULT_TIMEOUT = 20 and use across.

Reply:
done